ACMILabs/interactive-label
# #64 Update node packages

Resolves #64

Update node packages to resolve the `node-notifier` vulnerability.

### Acceptance Criteria
- [x] Update node packages to resolve the `node-notifier` vulnerability

### Relevant design files
* None

### Testing instructions
1. Run `npm install`
2. Run `npm list node-notifier` to see that it is v8.0.1
3. Run `docker exec -it javascripttests make linttestjs` to see testing and linting passes

### DoD
For requester to complete:
